src/libraries/Microsoft.Extensions.Configuration.UserSecrets/src/Microsoft.Extensions.Configuration.UserSecrets.csproj

```
@@ -4,8 +4,10 @@
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
     <PackageDescription>User secrets configuration provider implementation for Microsoft.Extensions.Configuration.</PackageDescription>
+    <!-- Disable package on release/6.0-maui branch, revert before merging back into release/6.0
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <ServicingVersion>1</ServicingVersion>
+    -->
```

revert the branch specific changes:
```suggestion
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <ServicingVersion>1</ServicingVersion>
```